scripts/bundlesize/bundlesize.mjs

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+// Simple script to check our bundlesize
+import fs from 'fs/promises';
+import { promisify } from 'util';
+import { gzip as gzipCallback } from 'zlib';
+import { fileURLToPath } from 'url';
+import path from 'path';
+import bytes from 'bytes';
+import { minify } from 'terser';
+
+const gzip = promisify(gzipCallback);
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const jsonFile = path.join(__dirname, 'bundlesize.config.json');
+const { files } = JSON.parse(await fs.readFile(jsonFile, 'utf-8'));
+
+let failed = false;
+
+for (const { path: filePath, maxSize } of files) {
+    const absoluteFile = path.join(__dirname, '../..', filePath);
+    const source = await fs.readFile(absoluteFile, 'utf-8');
+    const { code } = await minify(source, { compress: true, mangle: true });
```

Comment:
Why is it necessary to re-minify the `.min.js` files? (_They are minified using terser originally_)

Reply:
You are correct, my mistake. Fixed.